contributing/review-process.md

```diff
@@ -0,0 +1,47 @@
+# Respectful Changes and Code Reviews
+
+
+This document aims to provide clear guidelines and expectations for code authors contributing to The Falco Project and for reviewers performing code reviews.
+
+Drawing inspiration from the Chromium project, we aim to incorporate their exemplary resources and general guidance into our own practices.
+
+
+- [Chromium Respectful Changes](https://chromium.googlesource.com/chromium/src/+/main/docs/cl_respect.md)
+- [Respectful Code Reviews](https://chromium.googlesource.com/chromium/src/+/main/docs/cr_respect.md)
+
+
+In The Falco Project, we strive to foster a positive and constructive environment for the review process. Keeping this in mind, we would like to highlight the following points:
+
+
+## Guidelines for Code Authors
+
+> A special note for first-time contributors:   
+>
+> Please take the time to study and understand the project DNA by examining its history, workflow, and style. It is advisable to start with small patches and code changes to become familiar with the processes and build relationships and trust with the maintainers. If needed, do not hesitate to ask for help and feedback early on. We encourage everyone to work in the open.
+
+
+- Understand and respect Falco's [governance](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md).
+- Be mindful of your reviewers' time and make an effort to meet all preconditions, including thorough testing and providing sufficient explanations for your code changes. Additionally, consider dividing large and high-risk PRs into multiple smaller ones whenever possible.
+- The Falco maintainers are committed to performing a first-pass code review within a maximum of one week, with an ideal target of 24 hours. As maintainers and reviewers in The Falco Project are spread across different repositories, we strive to be as responsive as possible for each PR. However, depending on the complexity and scope, some reviews may take longer. As a code author, we encourage you to check the status and feel free to do so on a weekly basis. You can kindly post a comment in the respective PR saying, "I would like to check in on the status. Thank you." This proactive approach helps keep track of progress and facilitates effective communication.
+- In addition, certain reviews may require input from maintainers with specialized expertise in specific areas. This may result in some delays during the review process. We appreciate your understanding in such cases.
```

Comment:
```suggestion
- Additionally, certain reviews may require input from code reviewers with specialized expertise in specific areas. This may result in some delays during the review process. We appreciate your understanding in such cases.
```